modules/openapi-generator/src/main/resources/typescript/api/api.mustache

```diff
@@ -39,7 +39,7 @@ export class {{classname}}RequestFactory extends BaseAPIRequestFactory {
      * {{&summary}}
      {{/summary}}
      {{#allParams}}
-     * @param {{paramName}} {{description}}
+     * @param {{paramName}} {{{description}}}
```

Comment:
I noticed here, that we HTML-escape the description. We don't anywhere else, so I fixed this on the way, but it is only tangentially related to the pull request. Please let me know if you'd like to revert this. There are currently no sample fixtures which are covering this codepath.